src/lib/components/SegmentedControl.svelte

```diff
@@ -1,26 +1,19 @@
-<script lang="ts" context="module">
-	export interface SegmentedControlOption {
-		title: string;
-		value: string;
-	}
-</script>
-
 <script lang="ts">
 	import { createEventDispatcher } from 'svelte';
+	import type { SegmentedControlOption } from '$lib/types';
 
-	export let style: string | undefined = undefined;
-
+	export let style: string = undefined;
```

Comment:
The style can also not be defined, so why remove the optional `undefined` declaration.

Reply:
`undefined` should not be used as a type. If something is declared as `undefined` it is similar to a placeholder that indicates that the variable is yet to be declared. To explain this with a metaphor, we don't want `style` to be of type placeholder.

However, adding `undefined` is not wrong, it just adds overhead that is not needed.